tests/robustness/traffic/traffic.go

```diff
@@ -93,14 +93,17 @@ func SimulateTraffic(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2
 	}
 	reports = append(reports, cc.Report())
 
-	var operationCount int
+	var totalOperations int
+	var successfulOperations int
 	for _, r := range reports {
-		operationCount += len(r.KeyValue)
+		totalOperations += len(r.KeyValue)
+		successfulOperations += r.SuccessfulOperations()
 	}
-	lg.Info("Recorded operations", zap.Int("operationCount", operationCount))
+	lg.Info("Recorded operations", zap.Int("operations", totalOperations), zap.Float64("successRate", float64(successfulOperations)/float64(totalOperations)))
```

Comment:
Do you expect all the errors to be timeout errors? If not, would it make sense to collect failed counts by error types?

Reply:
No, categorizing errors would be next step, but I don't think we will need it to confirm that there are more errors on arm vs amd.